src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -116,9 +144,9 @@ public class NewAzureContainerGroupCommand : ContainerInstanceCmdletBase
 
         [Parameter(
             Mandatory = false,
-            HelpMessage = "The port to open. Default: 80")]
+            HelpMessage = "The ports to open. Default: [80]")]
         [ValidateNotNullOrEmpty]
-        public int? Port { get; set; }
+        public int[] Ports { get; set; }
```

Comment:
@yolo3301 why was this change made?

Reply:
We support multiple ports. In Azure CLI we have already made a similar change.

Reply:
@yolo3301 from PowerShell's _Strongly Encouraged Development Guidelines_ (the snippet can be found [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#singular-noun-1)), parameter names should be singular _unless_ there is validation for the parameter preventing the user from passing only one object. If a user can pass just one port, then this parameter should be `public int[] Port`

Reply:
Good to know. Updated.